src/mitol/openedx_user_retirement/exception.py

```diff
@@ -0,0 +1,9 @@
+"""Exceptions for openedx_user_retirement app"""
+
+
+class HttpDoesNotExistException(Exception):
+    """
+    Called when the server sends a 404 error.
+    """
+
+    pass
```

Comment:
Ideally, we should but I don't want to diverge a lot from Tubular itself so I think it's fine to keep it like that. At least this will serve as a marker for he retirement errors so that's good enough.